mirascope/core/base/_utils/_protocols.py

```diff
@@ -92,6 +93,10 @@ def __call__(
     ) -> _ResponseT | Generator[_ResponseChunkT, None, None]: ...
 
 
+# We can't set TypedDict as a bound for a TypeVar, so we use Mapping instead
+_BaseCallParamsT = TypeVar("_BaseCallParamsT", bound=Mapping, covariant=True)
```

Comment:
same question here, could you add some more explanation in this comment or the other one so I can understand this change better?

Reply:
Fixed